lib/SILOptimizer/Mandatory/TFDifferentiation.cpp

```diff
@@ -767,11 +768,14 @@ void DifferentiationInvoker::print(llvm::raw_ostream &os) const {
        << ") task=" << indDiff.second << ')';
     break;
   }
-  case Kind::DifferentialOperator:
+  case Kind::DifferentialOperator: {
+    StreamPrinter printer(os);
+    PrintOptions options;
     os << "differential_operator=(";
-    getDifferentialOperator()->print(os);
+    getDifferentialOperator()->print(printer, options);
```

Comment:
This might be suboptimal (importing `swift/AST/ASTPrinter.h` and init'ing `StreamPrinter` and `PrintOptions`).
Not sure which upstream commit made this change necessary. Without this change, I did get a compile-time crash.